NerdDinner/Models/PaginatedList.cs

```diff
@@ -1,29 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Web;
-
-namespace NerdDinner.Models
-{
-    public class PaginatedList<T> : List<T>
```

Comment:
I removed this because it wasn't being used anywhere. If it's part of your plan for the future, ignore me and keep it